tests/unit/schema/wrappers/test_method.py

```diff
@@ -289,6 +290,31 @@ def test_method_field_headers_present():
         assert method.field_headers == ('parent',)
 
 
+def test_method_routing_rule():
+    routing_rule = routing_pb2.RoutingRule()
+    param = routing_rule.routing_parameters.add()
+    param.field = 'table_name'
+    param.path_template = 'projects/*/{table_location=instances/*}/tables/*'
+    method = make_method('DoSomething', routing_rule=routing_rule)
+    assert method.explicit_routing
+    assert method.routing_rule.routing_parameters == [wrappers.RoutingParameter(
+        x.field, x.path_template) for x in routing_rule.routing_parameters]
+    assert isinstance(
+        method.routing_rule.routing_parameters[0], wrappers.RoutingParameter)
```

Comment:
I don't think this check is necessary; the check above tests equality, which should check the type since you haven't overridden `__equal__`.

Reply:
Done